test/spell_check.words

```diff
@@ -1,25 +1,24 @@
-app
+ament
+apache
 argcomplete
-args
-basepath
-bool
-buildfile
 colcon
-config
-cpu
-dict
+completers
 esteve
-fhs
+fernandez
 github
 gradle
 https
-jobserver
```

Comment:
Was this removed because it is no longer in the codebase or because it's in the upstream dictionary for scspell?

Reply:
I don't see `jobserver` appearing anywhere in this repository. I believe it was probably an artifact from copy/paste of another colcon extension.

Reply:
Note that this dictionary file is robustly checked by the test. It should exactly match what is necessary for the check to succeed - no more, no less, and in alphabetical order.